reference/functions/findfiles.markdown

```diff
@@ -17,14 +17,15 @@ match all files in one directory that end in `.cf` but it won't search
 across directories.  `*/*.cf` on the other hand will look two levels
 deep.
 * `?` matches a single letter
-* `[a-z]` matches any letter from `a` to `z`
+* `[a-z]` matches any letter from `a` to `z` (not yet supported on Windows)
```

Comment:
@larsewi do we have a ticket for adding this support?
- It should link to the docs here to update them when support is added.

Reply:
https://northerntech.atlassian.net/browse/CFE-4255

Reply:
ty